Externals/fmt/include/fmt/core.h

```diff
@@ -1702,24 +1702,24 @@ template <typename Context, typename T>
 FMT_CONSTEXPR FMT_INLINE auto make_value(T&& val) -> value<Context> {
   const auto& arg = arg_mapper<Context>().map(std::forward<T>(val));
 
-  constexpr bool formattable_char =
+  static constexpr bool formattable_char =
```

Comment:
`[[maybe_unused]]` instead of the `static` should probably do the trick.

Reply:
unfortunately doesn't seem to work.

Reply:
nvm, it does work, thanks. (was just using regex replace and missed that there's actually two different cases that need to be considered)